sceptre/plan/actions.py

```diff
@@ -848,6 +848,10 @@ def _log_new_events(self, after_datetime: datetime) -> datetime:
                         event["ResourceType"],
                         event["ResourceStatus"],
                         event.get("ResourceStatusReason", ""),
+                        event["HookType"],
+                        event["HookStatus"],
+                        event.get("HookStatusReason", ""),
+                        event["HookFailureMode"],
```

Comment:
I'd like to see examples of what the logs look like when there IS hook information and when there ISN'T hook information. Can you post some screenshots? I'm hesitant to add four new datapoints to every single log message if it's going to clutter the logs when 99% of events won't involve any hooks

Reply:
Hey @jfalkenstein 
Thank you for your time and effort. I tried to collect some screenshots for your reference. Please let me know if they help to make things clear. 

Please see below the No.1 screenshot. This is what I can see from Sceptre output, which was running in a Jenkins pipeline. 
![test-cf-hook-sceptre-01](https://github.com/Sceptre/sceptre/assets/56063981/218aada1-630c-4973-8471-29075cd50d01)

As I have configure the cloudformation hook mode to 'WARN', the stack was created without any blocking and I was thinking everythin works just fine.

However, if I double check the cloudformation events from aws web console for more details, I can see below:
![test-cf-hook-web-console-06](https://github.com/Sceptre/sceptre/assets/56063981/48b672f4-f703-4fce-b1b7-a2c47a91e488)

That was saying the hook's lambda function does not work at all.

If Sceptre can display more details about cloudformation hooks, such as 'HookStatusReason', I can pick up this issue immediately from my Jenkins pipeline console. Otherwise, I would think my hook is doing fine until I dig deeper in aws web console.

In short, I hope Sceptre can support this feature in next release please.

Thank you.
Feng